kokoro/config/build/presubmit/rockylinux9.gcl

```diff
@@ -0,0 +1,12 @@
+import '../common.gcl' as common
+
+config build = common.build {
+  params {
+    environment {
+      // centos9 is built ontop of rockylinux:9 in Dockerfile:
+      // https://github.com/GoogleCloudPlatform/ops-agent/blob/72488e91d2f5dbd5ba5dc8c30f493deb7802dc09/Dockerfile#L163
+      DISTRO = 'centos9'
```

Comment:
IMO, pending discusssion on team chat, we should not refer to centos9 at all, instead we should pass DISTRO = 'rockylinux9' and change the Dockerfile to refer to it as rockylinux9 as well. (Looks like igor agrees, let's use rockylinux9 instead).

calling it centos9 is confusing since IIUC CentOS 9 doesn't actually exist.

Reply:
Changed in: [8654fa1](https://github.com/GoogleCloudPlatform/ops-agent/pull/812/commits/8654fa10aa977e748769f77935aa2bd9dc07b352)